mlflow/metrics/genai/prompts/v1.py

```diff
@@ -83,41 +83,40 @@ def _format_examples(self):
 
 
 @dataclass
-class CorrectnessMetric:
+class AnswerSimilarityMetric:
     definition = (
-        "Correctness is evaluated on the proximity of the provided output to the ground truth "
-        "in terms of meaning and description similarity. Scores can be assigned from 1 to 5 based "
-        "on the gradual similarity in meaning and description to the ground truth."
+        "Answer similarity is evaluated on the degree of semantic similarity of the provided output"
+        " to the provided target, which is the ground truth. Scores can be assigned based on the "
+        "gradual similarity in meaning and description to the provided target, where a higher "
+        "score indicates greater alignment between the provided output and provided target."
     )
 
     grading_prompt = (
-        "Correctness: Below are the details for different scores:"
-        "- Score 1: the output is completely incorrect, doesn't mention anything related to the "
-        "input or is completely contrary to the provided ground truth."
-        "- Score 2: the output provides some relevance to the input and answers one aspect of the "
-        "question as in the ground truth."
-        "- Score 3: the output mostly answers the question but is missing or hallucinating on "
-        "one critical aspect."
-        "- Score 5: the output correctly answers the question and is not missing any major aspect "
-        "provided in the ground truth answer."
+        "Answer Similarity: Below are the details for different scores:"
+        "- Score 1: the output has little to no semantic similarity to the provided target."
+        "- Score 2: the output displays partial semantic similarity to the provided target on some "
+        "aspects."
+        "- Score 3: the output has moderate semantic similarity to the provided target."
+        "- Score 4: the output aligns with the provided target in most aspects and has substantial"
+        " semantic similarity."
+        "- Score 5: the output closely aligns with the provided target in all significant aspects."
     )
 
-    grading_context_columns = ["targets"]
+    grading_context_columns = ["target"]
```

Comment:
Shouldn't this still be `targets` ? mlflow.evaluate calls it targets: https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.evaluate

Reply:
Even here we expect the param name to be targets:
https://github.com/mlflow/mlflow/blob/98d619cf6327199f7d7302905a3b5a91b72c91a4/mlflow/models/evaluation/default_evaluator.py#L1143

Let me know if I am missing something,

Reply:
I wanted to change this to target because I felt like it makes less sense in the prompt for it to be targets than singular target - for instance we want to tell the LLM judge to compare the provided output to the provided target. And we input the ground truth as "Provided target: {ground_truth}" which doesn't really make sense to have "Provided targets: {only one thing}". Also for other metrics, the names for these grading_context_columns will all be singular (e.x. context, ground_truth, although we are not using ground_truth anymore). LMK if this doesn't make sense

Reply:
I guess to do that we could update the param_name check, but that is inconsistent with "targets=" in mlflow.evaluate() call that we should not change... Do we prefer to have targets consistently and have a less clear prompt

Reply:
Got it. That makes sense. 
We should surely change one or the other since with your current change correctness metric won't work. 
Let's get inputs from Corey and Prithvi as well.
My 2cents, let's do targets since the eval_fn accepts targets as well since we return back the entire column in eval_fn. 
So using that consistency with make_metric api makes sense to me. Thoughts?

Reply:
Yeah that makes sense, I can go in and modify the prompt to be consistent with this.

Reply:
+1, agree that we should modify the prompt and not the evaluate API.